modular_skyrat/modules/biohazard_blob/code/mold_event.dm

```diff
@@ -3,7 +3,7 @@
 	typepath = /datum/round_event/mold
 	weight = 5
 	max_occurrences = 1
-	min_players = 10
```

Comment:
We should still have minimum players on this, otherwise it'll spawn in lowpop and on test servers.